silverback/_cli.py

```diff
@@ -87,7 +100,7 @@ def _network_callback(ctx, param, val):
     callback=cls_import_callback,
 )
 @click.option("-x", "--max-exceptions", type=int, default=3)
-@click.argument("path")
+@click.option("-p", "--path", type=str, default=None)
```

Comment:
You can actually have default arguments
```suggestion
@click.argument("path", type=str, default=None)
```